### ISSUE TITLE
Can not peer with external BGP router

### ISSUE THREAD

**Issue by tiukhtinvladimir** [cloudnativelabs/kube-router#507]
```
# kube-router --peer-router-ips=192.168.1.1 --peer-router-asns=44252 --kubeconfig=/etc/kubernetes/kube-router.conf --advertise-loadbalancer-ip --advertise-external-ip --advertise-cluster-ip --run-firewall=true --run-router=true --run-service-proxy=true
I0807 13:13:06.408048   23784 kube-router.go:186] Running kube-router version , built on , go1.10
I0807 13:13:06.409729   23784 health_controller.go:145] Starting health controller
I0807 13:13:07.120020   23784 network_policy_controller.go:125] Starting network policy controller
Failed to run kube-router: Failed to create network routing controller: Error processing Global Peer Router configs: Invalid ASN number "44252" for global BGP peer
```

**Comment by murali-reddy** [comment]
@tiukhtinvladimir you need to use private ASN

```Failed to create network routing controller: Error processing Global Peer Router configs: Invalid ASN number "44252" for global BGP peer```

Or you wanted to actually use public ASN?

**Comment by tiukhtinvladimir** [comment]
My issue is that I can not place my Cisco routers in two AS - it is a Cisco restriction. So as far as they are in a real public AS already serving BGP I am trying to peer kube-routers with them as with public routers over eBGP.

**Comment by murali-reddy** [comment]
So right now ASN number is restricted to a value between 64512-65535 and 4200000000-4294967294 so not surprising that you are running in to this issue. Perhaps `error` could be translated to `warning`. I can make the change, PR is welcomed though.

```go
	if kubeRouterConfig.ClusterAsn != 0 {
		if !((kubeRouterConfig.ClusterAsn >= 64512 && kubeRouterConfig.ClusterAsn <= 65535) ||
			(kubeRouterConfig.ClusterAsn >= 4200000000 && kubeRouterConfig.ClusterAsn <= 4294967294)) {
			return nil, errors.New("Invalid ASN number for cluster ASN")
		}
		nrc.defaultNodeAsnNumber = uint32(kubeRouterConfig.ClusterAsn)
	} else {
		nrc.defaultNodeAsnNumber = 64512 // this magic number is first of the private ASN range, use it as default
	}
```

**Comment by rillion** [comment]
Hi, 

Could this change be made or atleast a flag to allow public ASN peers?

**Comment by rillion** [comment]
Also, 

It's actually coming from here:

pkg/controllers/routing/bgp_peers.go
`	for i := 0; i < len(ips); i++ {
		if !((asns[i] >= 64512 && asns[i] <= 65535) ||
			(asns[i] >= 4200000000 && asns[i] <= 4294967294)) {
			return nil, fmt.Errorf("Invalid ASN number \"%d\" for global BGP peer",
				asns[i])
		}`

**Comment by murali-reddy** [comment]
Fixed by #576
